course_catalog/tasks_helpers.py

```diff
@@ -0,0 +1,124 @@
+"""
+course_catalog helper functions for tasks
+"""
+import json
+
+from datetime import datetime
+
+import pytz
+import requests
+from django.db import transaction
+
+from course_catalog.constants import PlatformType, semester_mapping
+from course_catalog.models import Course, CourseTopic, CourseInstructor, CoursePrice
+from course_catalog.serializers import CourseSerializer
+from course_catalog.settings import EDX_API_CLIENT_ID, EDX_API_CLIENT_SECRET
+
+
+def get_access_token():
+    """
+    Get an access token for edx
+    """
+    post_data = {
+        "grant_type": "client_credentials",
+        "client_id": EDX_API_CLIENT_ID,
+        "client_secret": EDX_API_CLIENT_SECRET,
+        "token_type": "jwt"
+    }
+    response = requests.post("https://api.edx.org/oauth2/v1/access_token", data=post_data)
+    return response.json()["access_token"]
+
+
+def parse_mitx_json_data(course_data):
```

Comment:
I think it would be more consistent with our open-discussions repo to move a bunch of this code into an `MITxCourseSerializer`, similar to the `OCWCourseSerializer`, but the code works so it's fine for now.